samples/SocketsSample/wwwroot/hubs.html

```diff
@@ -130,7 +130,7 @@ <h1 id="head1"></h1>
 
 click('groupmsg', event => {
     let groupName = getText('target');
-    let message = getText('messages');
+    let message = getText('message');
```

Comment:
I wonder if it would be better to rename this. We have two ids - "message" and "messages" and they are being confused. I think this is the second or third PR trying to get things straight.